code/game/objects/structures/mineral_doors.dm

```diff
@@ -83,26 +83,22 @@
 		icon_state = "[base_icon_state][smoothing_flags ? "-[smoothing_junction]" : ""]"
 
 
-/obj/structure/mineral_door/attackby(obj/item/W, mob/living/user)
+/obj/structure/mineral_door/attackby(obj/item/attacking_item, mob/living/user)
 	. = ..()
 	if(.)
-		return
+		return TRUE
 	if(QDELETED(src))
 		return
+	if(istype(attacking_item, /obj/item/tool/pickaxe/plasmacutter) && !user.do_actions) //Special plasmacutter interaction
+		var/obj/item/tool/pickaxe/plasmacutter/pcutter = attacking_item
+		if(pcutter.start_cut(user, name, src, PLASMACUTTER_BASE_COST * PLASMACUTTER_VLOW_MOD))
+			pcutter.cut_apart(user, name, src, PLASMACUTTER_BASE_COST * PLASMACUTTER_VLOW_MOD) //Minimal energy cost.
+			user.changeNext_move(attacking_item.attack_speed)
+			user.do_attack_animation(src, used_item = attacking_item)
+			take_damage(max_integrity, BURN, FIRE, armour_penetration = 100)
```

Comment:
Why the change from a normal hit from the item with multipliers to just instant deletion, ignoring all armor? This also removes other burn-based melee weapons (small niche, admittedly) from having a bonus.

Reply:
Because plasma cutter instantly cuts wall, so it should cut doors as well.

Reply:
That is besides the point since this also affects things that arent a plasmacutter